src/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -13,16 +13,6 @@
     <DefineConstants Condition=" '$(IsInterpreting)' != 'true' ">$(DefineConstants);FEATURE_COMPILE</DefineConstants>
     <DefineConstants Condition=" '$(FeatureInterpret)' == 'true' ">$(DefineConstants);FEATURE_INTERPRET</DefineConstants>
   </PropertyGroup>
-  <ItemGroup>
-    <ReferenceFromRuntime Include="System.Private.CoreLib" />
```

Comment:
I wonder if you could get rid of the OS-specific targeting too?  I didn't see anywhere this different between Windows and Unix.

Reply:
I'll check.

Reply:
@marek-safar noticed this today.  I think we can remove it.